briskedu/brisk-learning
# Adding Submit feedback button

Summary:
Adding option to submit feedback

Will add email or slack post later.

https://app.asana.com/0/1203963408726580/1204221715968604

Test Plan:
1. Confirm I can submit feedback
2. Confirm it's in DB
3. Confirm confirmation link shows up and is closeable
